CharacterCreation/05_ArmourList.txt

```diff
@@ -524,7 +519,7 @@ Secondary Module Benefit: Either provides +60 shield strength
 OR +60 recharge rate, +4 fast recharge cost per secondary.
 Decided on purchase.

```

Comment:
These newlines in the end of the stat block are not going to roll into the line above it. I'm not sure what the best solution for this is.